### DISCUSSION TITLE
Let chose the output device in the programs/device volume list

### DISCUSSION THREAD

**Discussion by AndelsonMarcelino** [File-New-Project/EarTrumpet#1242] [Q&A] [answered]
When um you left click your mouse, a list of devices and volumes are showed, I can mute a specific program, change whole volume level, but a can't chance the output device. I would like to get it.

**Comment by riverar** [comment, marked as answer]
Hi @AndelsonMarcelino you can do that by right-clicking a row and left-clicking on the arrows:

![image](https://user-images.githubusercontent.com/475132/194370722-43c58bd2-01d7-4a35-bc40-5397c6d6749a.png)

**Comment by AndelsonMarcelino** [comment]
But then, it will be easier to right click, right over the ET in the task bar to do this. I would like to click over the device name, for exemple, and select the device. You Got it.
But, thank you, anyway.

My best regards

Att

Andélson Marcelino

Enviado do Email para Windows 11

De: Rafael ***@***.***>
Enviado:quinta-feira, 6 de outubro de 2022 13:42
Para: ***@***.***>
***@***.***>; ***@***.***>
Assunto: Re: [File-New-Project/EarTrumpet] Let chose the output device in the programs/device volume list (Discussion #1242)


Hi @AndelsonMarcelino<https://github.com/AndelsonMarcelino> you can do that by right-clicking a row and left-clicking on the arrows:

[image]<https://user-images.githubusercontent.com/475132/194370722-43c58bd2-01d7-4a35-bc40-5397c6d6749a.png>
